### ISSUE TITLE
Project is discontinued, looking for new mantainer

### ISSUE THREAD

**Issue by matteocrippa** [matteocrippa/flutter-nfc-reader#38]
If any want to continue can add to contributor or transfer of the project so can continue to manage that in the future

**Comment by huzhiren** [comment]
@matteocrippa You could let me join.

**Comment by f1re** [comment]
since i am using it productive soon, i forked it and fixed already some bugs and made some improvements on my fork

**Comment by matteocrippa** [comment]
@f1re if you want feel free to make a pr for that so also other can benefit of that

**Comment by xiaods** [comment]
@f1re are you sure the plugin is working? i use it and can't scan any nfc tag .

**Comment by xiaods** [comment]
@matteocrippa i currently working on nfc project, i can support this project.

**Comment by henrysachs** [comment]
I implemented something to scan multiple tags maybe you wanna chat about it and merge it back into the lib?

**Comment by matteocrippa** [comment]
hey @henrysachs, yep feel free to make a PR, so we can review with other maintainer

**Comment by henrysachs** [comment]
who is the new maintainer?

**Comment by matteocrippa** [comment]
still me for now, but few time to look at

**Comment by henrysachs** [comment]
yeah I dont have that much time either but think on using that package for a little project myself. Maybe we can do something on par because few time and few time is a little more time. :D

I think you can assume a PR from me tomorrow is that okay for you? :)

**Comment by huzhiren** [comment]
I used this package before, but it looks like not stable (only scan once and then stop working, tested on Samsung),   and is not working on ios,  then I switch to [nfc_io](https://pub.dev/packages/nfc_io), much stable so far.

**Comment by henrysachs** [comment]
Hey I can understand your situation but this will change soon. Maybe you keep an open eye on the plugin. :)

**Comment by huzhiren** [comment]
Yes, I understand that.  But the client don't care about that at all.

**Comment by matteocrippa** [comment]
@ryanhz tbh the only one scan was the expected behaviour, btw now with help of @henrysachs we are going to add unlimited scan support.

Hope see you coming back when customer will pay you for iOS version, feature that we have at least.

**Comment by ryanhz** [comment]
no, when i said only one scan, i meant it really scan once only for the lifetime of the app, the last time as I test, you need to disable nfc and re-enable nfc on the phone and restart the app to have it able to scan again.

**Comment by matteocrippa** [comment]
You know at the end, open source project are always open and welcome to improvements by anyone :)
